index.html

```diff
@@ -27,16 +26,177 @@ <h1 class="intro">I'm Morteza Glad to See You!</h1>
+
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+
+            <div class="techs">
+                <div class="tag">
+                    <a href="#">html</a>
+                </div>
+                <div class="tag">
+                    <a href="#">CSS</a>
+                </div>
+                <div class="tag">
+                    <a href="#">Javascript</a>
+                </div>
+            </div>
+            <div class="action">
+                <div class="btn"><a href="">See Project</a> </div>
+            </div>
+        </section>
+
+
+        <section class="about">
+            <div class="top">
+                <p class="myself">About Myself</p>
+                <p class="desc">Hello I’m a software developer! I can help you build a product , feature or website Look
+                    through some of my work and experience! If you like what you see and have a project you need coded,
+                    don’t hestiate to contact me.</p>
+                <div class="connect">LET'S CONNECT</div>
+                <div class="social">
+                    <ul class="social_nav">
+                        <li><a href="twitter.com/tawakulikhadim"><img src="assets/image_social/vitctor_t.svg"
+                                    alt="Twitter"></a>
+                        </li>
+                        <li><a href="https://github.com/tawakuliKH"><img src="assets/image_social/victor_g.svg"
+                                    alt="Github"></a></li>
+                        <li><a href=""><img src="assets/image_social/victor_h.svg" alt=""></a></li>
+                        <li><a href=""><img src="assets/image_social/victor_m.svg" alt="Mediam"></a></li>
+                        <li><a href=""><img src="assets/image_social/victor_l.svg" alt="Linked In"></a></li>
+                    </ul>
+
+                </div>
+                <div class="action">
+                    <div class="btn-resume"><a href="">Get My Resume</a> </div>
+                </div>
+            </div>
+
+            <div>
+
+                <ul class="main-nav">
+                    <li class="first"><a href="#" class="langueges">Langueges </a> <img src="assets/Icon - Chevron.png"
+                            alt="" width="20px" height="20px"></li>
+                    <li><a href="#"><img src="assets/js.png" alt=""></a></li>
+                    <li><a href="#"><img src="assets/html.png" alt=""></a></li>
+                    <li><a href="#"><img src="assets/css.png" alt=""></a></li>
+                    <li class="first"><a href="#" class="langueges">Frameworks </a> <img src="assets/Icon - Chevron.png"
+                            alt="" width="20px" height="20px"></li>
+                    <li class="first"><a href="#" class="langueges">Skills </a> <img src="assets/Icon - Chevron.png"
+                            alt="" width="20px" height="20px"></li>
+
+                </ul>
+
+            </div>
```

Comment:
- It will be great to stick to the design on Figma. Can we ensure that the Frameworks and skills have a border bottom just below each one of them? Currently, what you have on your website doesn't contain any of those.

- Can we kindly fix the spacing on the right side of the languages list section?  You can add some margin to it in order to create more space on the right.

Here is what i mean 👇🏽 
| Your Design  | How it should be (Figma Design) |
| :---:   | :---: | 
| <img width="380" alt="Screenshot 2023-01-20 at 15 34 10" src="https://user-images.githubusercontent.com/99246359/213811166-11abcf1e-5057-4691-8831-216d15c9ba44.png"> |<img width="394" alt="Screenshot 2023-01-20 at 15 34 53" src="https://user-images.githubusercontent.com/99246359/213811258-ded7ce09-52ee-47d7-9ac8-523e9e921e97.png">  |